core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -118,6 +118,7 @@ object Defaults {
   val AutoCreateTopicsEnable = true
   val MinInSyncReplicas = 1
   val MessageDownConversionEnable = true
+  val validApiVersions = ApiVersion.allVersionsArray
```

Comment:
Could we reference the `ApiVersion.allVersionsArray` directly below?  This is not really a default, but a constraint.